samples/NuGet.Config

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <clear />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <add key="dotnet6" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6/nuget/v3/index.json" />
```

Comment:
This will allow us to reference daily builds of the release/6.0-preview1 branch in samples:
```xml
<PackageReference Include="Microsoft.EntityFrameworkCore" Version="6.0.0-preview.1.*" />
```